edweiss-app/components/lectures/remotecontrol/modal.tsx

```diff
@@ -424,4 +425,97 @@ export const TimerSettingModal: ReactComponent<{
+};
+
+
+
+
+// ------------------------------------------------------------
+// -------------      Question Broadcast Modal    -------------
+// ------------------------------------------------------------
+
+export const QuestionBroadcastModal: ReactComponent<{
+    modalRef: React.RefObject<BottomSheetModalMethods>;
+    id: string;
+    courseId: string,
+    lectureId: string,
+    question: string;
+    username: string;
+    likes: number;
+    broadcasted: string,
+    setBroadcasted: React.Dispatch<React.SetStateAction<string>>;
+    onClose: () => void;
+}> = ({ modalRef, id, courseId, lectureId, question, username, likes, broadcasted, setBroadcasted, onClose }) => {
+
+    async function handleQuestionBroadcast() {
+        onClose();
+        if (broadcasted === id) {
+
+            // Question has been answered 
+            try {
+                await callFunction(LectureDisplay.Functions.markQuestionAsAnswered, {
+                    courseId: courseId,
+                    lectureId: lectureId,
+                    id: id,
+                    answered: true,
+                });
+                await callFunction(LectureDisplay.Functions.clearQuestionEvent, {
+                    courseId: courseId,
+                    lectureId: lectureId,
+                    id: id,
+                });
+            } catch (error) { console.error("Error updating the question event:", error); }
+
+            setBroadcasted("");
+
+        } else {
+
+            // Broadcast the question to the audience
+            try {
+                await callFunction(LectureDisplay.Functions.broadcastQuestion, {
+                    courseId: courseId,
+                    lectureId: lectureId,
+                    id: id,
+                });
+            } catch (error) { console.error("Error updating the question event:", error); }
+
+            setBroadcasted(id);
+        };
+    }
+
+    return (
+        <ModalContainer modalRef={modalRef}>
+            <>
+                <TView justifyContent='center' alignItems='center' mb='sm'>
+                    <TText bold size='lg' mb='sm'>{t('showtime:question_broadcast_modal_title')}</TText>
+                </TView>
+
+
+                <TText ml={'md'} color='overlay2' bold>{username === "" ? t('showtime:anony_ask_question') : username} {t('showtime:question_broadcast_modal_says')}</TText>
+
+
+                <TView justifyContent='center' alignItems='center' m={'md'}>
+                    <TText size={'lg'} color='overlay2' align='center'>« {question} »</TText>
+                </TView>
+
+
+                <TView flexDirection='column' alignItems='flex-end' mt='md'>
+                    {likes > 0 && (
+                        <>
+                            <TText ml={'md'} color='overlay2' mr='lg'>{likes} {t('showtime:other_student')}</TText>
+                            <TText ml={'md'} color='overlay2' mr='lg'>{t('showtime:are_interrested')}</TText>
+                        </>
```

Comment:
You use the same styles in many place, you can centralize them.

Reply:
Alright, I did not knew we could do this, can you tell me how to do this I am really interrested in it. 
Note, I prefer pass throw the `ml` .., arguments rather than overiding the style one. Because this way we use the `TText` interface, and stays consistent accross the entire codebase.

Reply:
I propose this : 


```
const styles = StyleSheet.create({
  overlayInfo: {
    marginLeft: 'md',
    marginRight: 'lg',
    color: 'overlay2',
  },
});

<>
  <TText style={styles.overlayInfo}>{likes} {t('showtime:other_student')}</TText>
  <TText style={styles.overlayInfo}>{t('showtime:are_interrested')}</TText>
</>
```

Reply:
Okay that's indeed a great solution to this. The only pitfall is that we need to pass throw the style  as mentioned above. I will try to find another solution that can reuse the predefined fields. If I do not I will go with your patch.